demos/formula_recognition_demo/python/README.md

```diff
@@ -35,6 +35,8 @@ The demo application takes an input with the help of the `-i` argument. This cou
 * Path to a video (.avi, .mp4, etc)
 > This will trigger interactive mode, which would be explained in detail later.
 
+Vocabulary files are provided under corresponding model description folder.
```

Comment:
```suggestion
Vocabulary files are provided under corresponding model configuration directory.
```

I think "description" can be ambiguous, because there is a subdirectory literally called `description`, but that's not the one you want. This directory is sometimes referred to as `config_dir` in `model.yml` files, so I think "configuration directory" makes the most sense.

Reply:
agree

Reply:
Please update the other occurrences, as well.